payjoin-cli/Cargo.toml

```diff
@@ -40,6 +40,7 @@ serde = { version = "1.0.160", features = ["derive"] }
 tokio = { version = "1.12.0", features = ["full"] }
 ureq = "2.9.4"
 url = { version = "2.3.1", features = ["serde"] }
+payjoin-io = { path = "../payjoin-io" }
```

Comment:
:+1: This makes a lot of sense to me vs a versioned dependency before our first release. I think we'll have to version it in order to release payjoin-cli with a payjoin-io dependency

Reply:
Since at this commit `payjoin-io` depends on `payjoin` with `v2` feature, It should also only be optionally compiled

```
v2 = ["payjoin/v2", "payjoin-io"]
...
payjoin-io = { path = "../payjoin-io", optional = true }
```

And included in the `v2` `payjoin-cli` feature